src/main/java/ru/promo/consul_plan/controller/NotificationController.java

```diff
@@ -1,64 +1,62 @@
 package ru.promo.consul_plan.controller;
 
 import jakarta.validation.Valid;
-import org.springframework.beans.factory.annotation.Autowired;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
-import ru.promo.consul_plan.dto.NotificationDTO;
-import ru.promo.consul_plan.entity.NotificationEntity;
+import ru.promo.consul_plan.domain.entity.NotificationEntity;
 import ru.promo.consul_plan.service.ConsultationService;
 import ru.promo.consul_plan.service.NotificationService;
 
 import java.util.List;
 
+@Slf4j
 @RestController
-@RequestMapping("/api/notifications")
-public class NotificationController {
+@RequiredArgsConstructor
+public class NotificationController implements NotificationApi{
 
-    @Autowired
-    private NotificationService notificationService;
-    @Autowired
-    private ConsultationService consultationService;
-
-
-    @PostMapping
+    private final NotificationService notificationService;
+    private final ConsultationService consultationService;
+    @Override
     public ResponseEntity<NotificationEntity> createNotification(@Valid @RequestBody NotificationEntity notification) {
```

Comment:
```suggestion
    public ResponseEntity<NotificationEntity> createNotification(NotificationEntity notification) {
```
эти аннотации должны быть в интерфейсе